modin/core/execution/client/query_compiler.py

```diff
@@ -356,15 +356,19 @@ def clip(self, lower, upper, **kwargs):
         if lower_is_qc:
             lower = lower._id
 
-    def isin(self, values):
+    def isin(self, values, values_is_series: bool, self_is_series: bool):
         # isin is unusal because it passes API layer objects to query compiler
         # instead of converting them to query compiler objects (Modin issue #3106)
-        from modin.pandas import Dataframe, Series
+        from modin.pandas import DataFrame, Series
```

Comment:
Interesting...is `Dataframe` even a thing in modin.pandas??

Reply:
Nope, but there's no tests for any client side changes, so this slipped through